src/content/configuration/devtool.md

```diff
@@ -32,34 +32,50 @@ T> The webpack repository contains an [example showing the effect of all `devtoo
+`nosources-cheap-module-source-map`        | build: slow<br>rebuild: slow       | yes        | original lines | source code not included
+`nosources-source-map`                     | build: slowest, rebuild: slowest   | yes        | original       | source code not included
+`hidden-nosources-cheap-source-map`        | build: ok<br>rebuild: slow         | yes        | transformed    | no reference, source code not included
+`hidden-nosources-cheap-module-source-map` | build: slow<br>rebuild: slow       | yes        | original lines | no reference, source code not included
+`hidden-nosources-source-map`              | build: slowest<br>rebuild: slowest | yes        | original       | no reference, source code not included
+`hidden-cheap-source-map`                  | build: ok<br>rebuild: slow         | yes        | transformed    | no reference
+`hidden-cheap-module-source-map`           | build: slow<br>rebuild: slow       | yes        | original lines | no reference
+`hidden-source-map`                        | build: slowest, rebuild: slowest   | yes        | original       | no reference. Possible choice when using SourceMap only for error reporting purposes.
+
+shortcut                  | explanation
+------------------------- | -----------
+performance: build        | How is the performance of the initial build affected by the devtool setting?
+performance: rebuild      | How is the performance of the incremental build affected by the devtool setting? Slow devtools might reduce development feedback loop in watch mode. The scale is different compared to the build performance, as one would expect rebuilds to be faster then builds.
+production                | Does it make sense to use this devtool for production builds? It's usually `no` when the devtool has a negative effect on user experience.
+quality: bundled          | You will see all generated code of a chunk in a single blob of code. This is the raw output file without any devtooling support
+quality: generated        | You will see the generated code, but each module is shown as separate code file in browser devtools.
+quality: transformed      | You will see generated code after the preprocessing by loaders but before additional webpack transformations.
+quality: original lines   | You will see the original code that you wrote, assuming all loaders support SourceMapping. Only source lines will be mapped and column information will be discarded resp. not generated. This prevents setting breakpoints in the middle of lines.
+quality: original         | You will see the original code that you wrote, assuming all loaders support SourceMapping.
+`eval-*` addition         | generate SourceMap per module and attach it via eval. Recommended for development, because of improved rebuild preformance. Note that there is a windows defender issue, which causes huge slowdown due to virus scanning.
```

Comment:
```suggestion
`eval-*` addition         | generate SourceMap per module and attach it via eval. Recommended for development, because of improved rebuild performance. Note that there is a windows defender issue, which causes huge slowdown due to virus scanning.
```